its/ruling/src/test/resources/expected/python-S930.json

```diff
@@ -28,7 +28,16 @@
 'project:twisted-12.1.0/doc/historic/2003/pycon/deferex/deferex-listing0.py':[
 4,
 ],
+'project:twisted-12.1.0/twisted/conch/manhole.py':[
+285,
+],
 'project:twisted-12.1.0/twisted/lore/slides.py':[
 260,
 ],
+'project:twisted-12.1.0/twisted/persisted/aot.py':[
+224,
+],
+'project:twisted-12.1.0/twisted/python/htmlizer.py':[
+81,
```

Comment:
Those are all somewhat TPs as the tokenize methods expects 
 now 1 argument. 2 arguments were needed only in python 2.7